src/services/eucovidcertService.ts

```diff
@@ -64,7 +65,7 @@ export default class EUCovidCertService {
           case 403:
             return ResponseErrorNotFound("Not Found", "Certificate not found");
           case 500:
-            return ResponseErrorInternal("");
+            return ResponseErrorInternal(readableProblem(response.value));
           case 504:
             return ResponseErrorInternal("");
```

Comment:
Doesn't is any error message detail for this response.

Reply:
there is now that I merged the fn-eucovidcert PR I mentioned ( pagopa/io-functions-eucovidcerts#26 )

Reply:
Done in [c1258c4](https://github.com/pagopa/io-backend/pull/808/commits/c1258c46d0f4d17961d69482e74ed7547a9d5c85)